seq_logger.js

```diff
@@ -80,27 +83,34 @@ class SeqLogger {
         this._batchSizeLimit = currentBatchSizeLimit;
         this._eventSizeLimit = currentEventSizeLimit;
 
-        const {dataParts, options, beaconUrl, size} = this._prepForBeacon(dequeued);
+        const { dataParts, options, beaconUrl, size } = this._prepForBeacon(dequeued);
 
         const data = new Blob(dataParts, options);
         return navigator.sendBeacon(beaconUrl, data);
     }
 
-    // Flush then destroy connections, close the logger, destroying timers and other resources.
-    close() {
+    /**
+     * Flush then destroy connections, close the logger, destroying timers and other resources.
+     * @returns {Promise<void>}
+     */
+    close () {
         if (this._closed) {
             throw new Error('The logger has already been closed.');
         }
-        
+
         this._closed = true;
         this._clearTimer();
         return this.flush().then(() => {
             this._httpAgent.destroy();
         });
```

Comment:
```suggestion
        return this.flush().then(() => {
            return this._httpAgent.destroy(); // <-- should this be returned? other, `close` returns `Promise<void>`
        });
```

Reply:
As far as I can tell, `destroy()` returns `void` - so probably not, here